### DISCUSSION TITLE
Ubuntu 22.04.4 LTS OvenMediaEngine

### DISCUSSION THREAD

**Discussion by Pixel-47** [OvenMediaLabs/OvenMediaEngine#1552] [Help]
Hello, I've been working on this for a few hours and haven't found a solution, but I searched and checked what's in the Server.xml file and by default it seems that everything is set correctly, unless I don't know where to edit something. I also did not find any option to access the web interface via a browser, which I have seen in other programs of this type. I'm adding a photo, unfortunately I can only see it on the laptop screen, I can't capture it in any other way. I use putty remotely and turn it on or off with the commands: sudo systemctl stop ovenmediaengine, sudo systemctl start ovenmediaengine, sudo systemctl status ovenmediaengine.
I installed this on Ubuntu 22.04.4 LTS

I use Tutorial : https://airensoft.gitbook.io/ovenmediaengine/v/dev/getting-started

**Comment by Pixel-47** [comment]
![1710419753107](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/ae32340a-9ce3-4a1e-b6fb-77e247272efd)

**Reply by bchah** [reply]
Looks like a regular OME startup to me. I think the issue is that you are looking for a web interface which does not exist 😅 (at least not in the public version of OME)

**Reply by Pixel-47** [reply]
When I stream, I get messages like in the picture, and everything is set by default. I also couldn't find a way to reproduce this image via the website.

**Reply by Pixel-47** [reply]
I mean the image of what I'm streaming.

**Reply by bchah** [reply]
The easiest way to view an OME stream in the browser is using [OvenPlayer](www.ovenplayer.com) or a number of supported HLS players.

**Comment by Pixel-47** [comment]
![image_instalation](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/06cc4179-c4ce-4a76-8c04-4fd9877fdfdf)
![image_Server-xml](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/e5c9ad79-c135-4daa-ab1a-6359582d690c)
![e66892b0-fa1c-49d0-b469-621e07e673e7when start](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/d5eb742f-17be-4972-a83c-c55450a24709)
I didn't change anything in the settings, I can stop and run the program, it starts with Linux, RTMP and SRT connects but the console shouts something. Does anything need to be set?

**Comment by Pixel-47** [comment]
![stream srt stop in obs studio](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/a769d179-2143-46be-acf2-a62381756a02)
This is what it looks like as soon as I stop the stream in OBS Studio via SRT.

**Reply by bchah** [reply]
Please provide your Server.xml and OME logs for analysis.

**Comment by Pixel-47** [comment]
[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/14606566/ovenmediaengine.log)
[Server.zip](https://github.com/AirenSoft/OvenMediaEngine/files/14606602/Server.zip)

I had to pack .xml because the website did not accept this .xml

**Reply by bchah** [reply]
Your logs show some streams starting, running for a while then either stopping or rebooting OME.

What I don't see are any playback sessions. How are you playing back (or trying to play back) the stream?

**Reply by Pixel-47** [reply]
I haven't figured out how to play it yet, I thought it could be opened by providing the server address and port.

**Comment by Pixel-47** [comment]
![this](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/46212496-fed0-498b-aaea-1ca5f5dbfc41)
I don't know why it's like this in this photo, that everything is turned off in cofig, and yet I can't find anything like that there, it's possible that something isn't set there because it's the same as it was created after installation. The description and guide on the website are very limited and it is difficult to match the path to this server.xml
So far, I haven't had any problems with .xml because I encountered these files in the DayZ game server, but I'm stuck here, it's hard for me to refer to the examples in the description and find the place from which this setting starts.

**Comment by Pixel-47** [comment]
I tested Owncast, AntMedia, Restreamer, even a tutorial with NGINX with RTMP from https://www.youtube.com/@ustoopia

But OvenMediaEngine is complicated, there are no guides that will help you on YouTube, there is nothing on the websites, only opinions of people who just give up after meeting this program.

**Reply by Pixel-47** [reply]
![ovnmediaengine status](https://github.com/AirenSoft/OvenMediaEngine/assets/154587581/ab588145-bd8a-4e69-9445-0830bacb7bdb)
sudo systemctl status ovenmediaengine
confirms that nothing is probably running in the config.

**Reply by getroot** [reply]
Those are not errors. You are using it normally. The logs you see mean that add-ons like record/restream are turned off in the settings, and the LLHLS publisher and WebRTC publisher for basic playback are working fine. Try WebRTC or LLHLS playback in OvenPlayer. You are doing great.
